lib/pbkit/pbkit.c

```diff
@@ -2018,7 +2019,7 @@ void pb_target_extra_buffer(int index_buffer)
         //DMA channel 10 is used by GPU in order to render depth stencil
         if (depth_stencil)
         {
-            dma_addr=pb_DSAddr&0x03FFFFFF;
+            dma_addr=0;//pb_DSAddr&0x03FFFFFF;
```

Comment:
Accidental change - that belongs with other DMA object change.
This is actually important as we'll have to fixup the surface offset then!